zerver/tests/test_notification_data.py

```diff
@@ -212,6 +216,43 @@ def test_is_notifiable(self) -> None:
         user_data = self.create_user_notifications_data_object(user_id=user_id, pm_push_notify=True)
         self.assertTrue(user_data.is_notifiable(acting_user_id=acting_user_id, idle=True))
 
+    def test_bot_user_notifiability(self) -> None:
+        class function_arguments(TypedDict):
```

Comment:
Here’s the [developer community thread](https://chat.zulip.org/#narrow/stream/92-learning/topic/mypy.3A.20Reusing.20arguments.20for.20function.20calls/near/1299681) where you asked about reducing the verbosity of this `TypedDict` construction. I think a [wrapper function](https://chat.zulip.org/#narrow/stream/92-learning/topic/mypy.3A.20Reusing.20arguments.20for.20function.20calls/near/1300149) or a [`for` loop](https://chat.zulip.org/#narrow/stream/92-learning/topic/mypy.3A.20Reusing.20arguments.20for.20function.20calls/near/1300561) would be a simpler solution.

Reply:
Thanks. I liked the `for` loop approach, and have pushed the same here.